Script/PRONTO.py

```diff
@@ -231,7 +232,123 @@ def clear_blank_line(file_in,file_out):
+			if(sheet_material.cell_value(r,c) == col_age):
+				ipd_age = str(sheet_material.cell_value(r+2,c))
+			if(sheet_material.cell_value(r,c) == col_consent and ipd_consent == ""):
+				ipd_consent = str(sheet_material.cell_value(r+2,c))
+				for r in range(r,(sample_info_row-2)):
+					if(ipd_consent == "0.0"):
+						ipd_consent = ""
+					if(sheet_material.cell_value(r,6) == col_requisition_hospital and requisition_hospital == ""):
+						requisition_hospital = sheet_material.cell_value(r+2,6)
+					if((sheet_material.cell_value(r+2,c) != "" or sheet_material.cell_value(r+2,c) != "-" or sheet_material.cell_value(r+2,c) != "0.0") and str(sheet_material.cell_value(r+2,c)) not in ipd_consent):
+						if(ipd_consent == ""):
+							ipd_consent = str(sheet_material.cell_value(r+2,c))
+						else:
+							ipd_consent = ipd_consent + "," + str(sheet_material.cell_value(r+2,c))
+						continue
+			if(sheet_material.cell_value(r,c) == col_material_name and ipd_material_id == ""):
+				for r in range(r,(extra_data_row-2)):
+					if(sheet_material.cell_value(r+2,9) == DNA_sampleID and sheet_material.cell_value(r+2,c) != "" and str(sheet_material.cell_value(r+2,c)) not in DNA_material_id):
+						if(DNA_material_id == ""):
+							DNA_material_id = str(sheet_material.cell_value(r+2,c))
+						else:
+							DNA_material_id = DNA_material_id + "," + str(sheet_material.cell_value(r+2,c))
+						tumor_content_nr = sheet_material.cell_value(r+2,2)
+						continue
+					if(RNA_sampleID != "" and sheet_material.cell_value(r+2,9) == RNA_sampleID and sheet_material.cell_value(r+2,c) != "" and str(sheet_material.cell_value(r+2,c)) not in RNA_material_id):
+						if(RNA_material_id == ""):
+							RNA_material_id = str(sheet_material.cell_value(r+2,c))
+						else:
+							RNA_material_id = RNA_material_id + "," + str(sheet_material.cell_value(r+2,c))
+						continue
+			if(sheet_material.cell_value(r,c) == col_extraction_hospital and extraction_hospital == ""):
+				for r in range(r,(library_pre_row-2)):
+					if(sheet_material.cell_value(r+2,8) == DNA_sampleID):
+						extraction_hospital = str(sheet_material.cell_value(r+2,c))
+						break
+			if(sheet_material.cell_value(r,c) == col_batch_nr and batch_nr == ""):
+				for r in range(r,(nrows_material-2)):
+					if(sheet_material.cell_value(r+2,0) == DNA_sampleID):
+						batch_nr = str(sheet_material.cell_value(r+2,c))
+	open_exl_material.release_resources()
+	if(ipd_consent == "0.0"):
+		ipd_consent = ""
+	if(ipd_age == "" and ipd_birth_date != ""):
+		ipd_age = "<1"
+	inclusion_site_list = {'R': 'Radium', 'U': 'Ullevål', 'C': 'Riksen', 'A': 'Ahus', 'D': 'Drammen', 'B': 'Bærum', 'G': 'Gjøvik', 'I': 'Hamar', 'L': 'Lillehammer', 'T': 'Vestfold', 'K': 'Sørlandet', 'Q': 'Østfold', 'V': 'Telemark', 'Y': 'Lovisenberg', 'H': 'Haukeland', 'S': 'Stavanger', 'E': 'Fonna', 'F': 'Førde', 'O': 'St.Olavs', 'M': 'Nord-trøndelag', 'J': 'Møre og Romsdal', 'N': 'Nord Norge', 'P': 'Nordland'}
```

Comment:
would it make sense to extract this information and add it to the config or similar? This way the information could be found more easily and edited if needed. 🙂

Reply:
Good suggestion! Maybe this could be a separate issue in the GitHub?

Reply:
of course, should I create an issue or will you?

Reply:
I could create an issue for this. :)